stage6/06-cosmetic/00-run.sh

```diff
@@ -6,4 +6,9 @@ cp -f files/launch.png "${ROOTFS_DIR}/usr/share/raspberrypi-artwork"
 on_chroot << EOF
 sed -i 's+^wallpaper=.*$+wallpaper=/usr/share/rpd-wallpaper/wallpaper.png+g' /etc/xdg/pcmanfm/LXDE-pi/desktop-items-0.conf
 sed -i 's+start-here+/usr/share/raspberrypi-artwork/launch.png+g' /etc/xdg/lxpanel/LXDE-pi/panels/panel
+
+#disable screensaver
+sed -i 's+@xscreensaver -no-splash+#@xscreensaver -no-splash+g' /etc/xdg/lxsession/LXDE-pi/autostart
+echo "@xset s off" >> /etc/xdg/lxsession/LXDE-pi/autostart
+echo "@xset -dpms" >> /etc/xdg/lxsession/LXDE-pi/autostart
```

Comment:
this is one of those things where some people would ask us to turn it back on;
but let's see;